### PR TITLE
Testing with JDBC as the event store

### PR DESCRIPTION
**Pull request by chbatey**
Only run locally with a local postgres and could push it to 1000 events/s through the projection and written to the read side view.

This goes up to 1400/s with the grouped projection

This also includes testing with AWS RDS and Astra Cassandra

### CODE REVIEW

**Comment by chbatey** [comment]
Failure s the pull limit

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -133,14 +137,64 @@ akka.persistence.cassandra {
...
+
+# the akka-persistence-snapshot-store in use
+jdbc-snapshot-store {
+  use-shared-db = "slick"
+}
+
+# the akka-persistence-query provider in use
+jdbc-read-journal {
+  use-shared-db = "slick"
+}
```
Comment (patriknw) [inline src/main/resources/application.conf]: would be nice to move to the same config structure as in the guide examples: https://akka.io/platform-guide/wip/microservices-tutorial/dev-env.html#_downloadable_example_source

**Review by patriknw — COMMENTED** [review]
looking good

File: src/main/resources/application.conf (hunk 1 above)
Comment (chbatey) [reply src/main/resources/application.conf]: Done! And integrated with the operator

File: README.md [hunk 2]
```diff
@@ -82,6 +82,61 @@ This should only happen in when multiple nodes are writing events as delayed eve
...
+## Deployment to EKS/GKE
+
+The Akka platform operator can be used to deploy this application to EKS for testing. 
+
+### Deploying to a new cluster with terraform
+
+Configure you aws client `aws configure`
+
+```
+cd terraform
```
Comment (chbatey) [inline README.md:94]: I did this with terraform as there are quite a lot of manual steps for EKS/RDS and i want to be able to tear this down over the weekend

File: README.md [hunk 3]
```diff
@@ -82,6 +82,61 @@ This should only happen in when multiple nodes are writing events as delayed eve
 order.
 
 
+## Deployment to EKS/GKE
+
+The Akka platform operator can be used to deploy this application to EKS for testing. 
+
+### Deploying to a new cluster with terraform
+
+Configure you aws client `aws configure`
```
Comment (octonato) [inline README.md]:
```suggestion
Configure your aws client with `aws configure`
```

File: src/main/resources/application.conf [hunk 4]
```diff
@@ -1,4 +1,4 @@
-akka.http.server.request-timeout = 600 s
+alocalhostkka.http.server.request-timeout = 600 s
```
Comment (octonato) [inline src/main/resources/application.conf]: mistake? 
```suggestion
akka.http.server.request-timeout = 600 s
```

**Review by octonato — COMMENTED** [review]
looking good

Spotted some small glitches, see below

File: README.md [hunk 5]
```diff
@@ -8,80 +8,135 @@ provide exactly once delivery to the projection.
...
 
-Each persistent actor is responsible for persisting the number of events, one at a time, it is instructed to. This means the request to persist
+each persistent actor is responsible for persisting the number of events, one at a time, it is instructed to. this means the request to persist
 the events can be retried meaning that even with failures to the messages table the test will eventually persist the right number of events.
 
-### Journal cleanup
+### journal cleanup
 
-Before every test the `messages` and `tag_views` test are truncated. Meaning when investigating failures the only messages in these tables
+before every test the `messages` and `tag_views` test are truncated. meaning when investigating failures the only messages in these tables
```
Comment (patriknw) [inline README.md]: what happened with the upper case?

File: src/main/resources/application.conf [hunk 6]
```diff
@@ -116,7 +89,7 @@ akka.persistence.cassandra {
   events-by-tag {
     bucket-size = "Hour"
     # for reduced latency
-    eventual-consistency-delay = 500ms
+    eventual-consistency-delay = 250
```
Comment (patriknw) [inline src/main/resources/application.conf]: 250ms

File: src/main/resources/local.conf [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    roles = ["write-model", "read-model"]
+  }
+
+  # For the sample, just bind to loopback and do not allow access from the network
+  # the port is overridden by the logic in main class
+  remote.artery {
+    canonical.port = 0
+    canonical.hostname = 127.0.0.1
+  }
+}
```
Comment (patriknw) [inline src/main/resources/local.conf]: \n

**Review by patriknw — COMMENTED** [review]
looking very good

File: src/main/resources/application.conf (hunk 4 above)
Comment (chbatey) [reply src/main/resources/application.conf]: Si señor

File: README.md (hunk 5 above)
Comment (chbatey) [reply README.md]: Doh

**Comment by chbatey** [comment]
I broke the int test, will fix that now and get it merged

**Comment by chbatey** [comment]
@patriknw / @octonato I'd like to merge this once travis is happy. It isn't perfect but we can iterate on it next time we want to run some tests.

**Pull request merged**